crates/sui-indexer/src/lib.rs

```diff
@@ -64,6 +64,8 @@ pub struct IndexerConfig {
     pub rpc_server_url: String,
     #[clap(long, default_value = "9000", global = true)]
     pub rpc_server_port: u16,
+    #[clap(long, multiple_occurrences = false, multiple_values = true)]
+    pub migrated_methods: Vec<String>,
```

Comment:
question: sorry missing some context here, what are `migrated_methods`?

Reply:
migrated method uses indexer native impl. while others are forwarded to FN.

we want to validate data returned by FN / indexer RPC endpoints, when the data validated, we can explicitly add endpoints to migrated methods via ops change